product-service/serverless.yml

```diff
@@ -25,14 +25,23 @@ provider:
   runtime: nodejs12.x
   stage: dev
   region: eu-west-1
+  environment:
+    PGHOST: ${file(./secret.json):PGHOST}
+    PGUSER: ${file(./secret.json):PGUSER}
+    PGDATABASE: ${file(./secret.json):PGDATABASE}
+    PGPASSWORD: ${file(./secret.json):PGPASSWORD}
+    PGPORT: ${file(./secret.json):PGPORT}
```

Comment:
just move it to .env file and use sls dotenv plugin

Reply:
It's great, thank you! Fixed